### PR TITLE
Mathlib usage howto and cleanup

### PR DESCRIPTION
**Pull request by irmen**
The code example that I've added works with 64tass and assembles without error with ca65 however I can't get a runnable program out of the latter. I assume the code works correctly for ca65 too, maybe someone can verify this. *edit* managed to get it to run correctly.

### CODE REVIEW

File: X16 Reference - 05 - Math Library.md [hunk 1]
```diff
@@ -5,85 +5,178 @@ The Commander X16 contains a floating point Math library with a precision of 40
...
-| C128  | C65   | X16   | Symbol   | Description                            |
-|-------|-------|-------|----------|----------------------------------------|
-| \$AF00 | \$7F00 | \$FE00 | `AYINT`  | convert floating point to integer      |
-| \$AF03 | \$7F03 | \$FE03 | `GIVAYF` | convert integer to floating point      |
-| \$AF06 | \$7F06 | \$FE06 | `FOUT`   | convert floating point to ASCII string |
-| \$AF09 | \$7F09 | \$FE09 | `VAL_1`  | convert ASCII string to floating point<br/>*[not yet implemented]*|
-| \$AF0C | \$7F0C | \$FE0C | `GETADR` | convert floating point to an address   |
-| \$AF0F | \$7F0F | \$FE0F | `FLOATC` | convert address to floating point      |
+| Address | Symbol   | Description                                                        |
+|---------|----------|--------------------------------------------------------------------|
```
Comment (mooinglemur) [inline X16 Reference - 05 - Math Library.md:13]: The content looks good, but the `$` symbols need to be escaped unless they're either inside an HTML table or inside a fenced code block. GitHub shows them correctly anyway, but the PDF creation has a more strict syntax.

**Review by mooinglemur — CHANGES_REQUESTED** [review]

File: X16 Reference - 05 - Math Library.md (hunk 1 above)
Comment (irmen) [reply X16 Reference - 05 - Math Library.md:13]: I've added the escaping back in. There's still an issue with the attempt to get the TeX formatted formula into the PDF

**Review by mooinglemur — APPROVED** [review]
Thanks! I will take a look at the PDF generation soon to see if we can get that working with the TeX

**Comment by irmen** [comment]
We can always just revert to the regular notation for the formula, it's just a nice to have in the PDF like this

**Pull request merged**